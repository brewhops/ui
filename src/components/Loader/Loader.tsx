import React from 'react';
import './Loader.styl';

export const Loader: React.FC = () =>   
<div className="lds-ring">
  <div></div>
  <div></div>
  <div></div>
  <div></div>
</div>