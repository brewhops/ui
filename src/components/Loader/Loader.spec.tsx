import React from 'react';
import { Loader } from './Loader';
import { shallow } from 'enzyme';

describe('src/components/Loader.tsx', () => {
  it('renders the loader', () => {
    expect(shallow(<Loader/>)).toMatchSnapshot();
  })
});