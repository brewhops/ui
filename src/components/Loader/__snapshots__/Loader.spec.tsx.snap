// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Loader.tsx renders the loader 1`] = `ShallowWrapper {}`;
